packages/slice-machine/src/features/sync/components/AutoSyncStatusIndicator.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC } from "react";
+import { Text, Tooltip } from "@prismicio/editor-ui";
+
+import { Syncing } from "@src/icons/Syncing";
+import { Synced } from "@src/icons/Synced";
+import { SyncFailed } from "@src/icons/SyncFailed";
+
+import styles from "./AutoSyncStatusIndicator.module.css";
+
+type AutoSyncStatusIndicatorProps = {
+  autoSyncStatus: "syncing" | "synced" | "failed";
+};
+
+export const AutoSyncStatusIndicator: FC<AutoSyncStatusIndicatorProps> = (
+  props,
+) => {
+  const { autoSyncStatus } = props;
+
+  let autoSaveStatusInfo;
+
+  console.log("autoSyncStatus:", autoSyncStatus);
```

Comment:
!

Reply:
Removed, console.log for QA on the alpha